### ISSUE TITLE
Pyinstaller issue while packaging spacy code

### ISSUE THREAD

**Issue by shikher04** [explosion/spaCy#6473]
exec(bytecode, module.__dict__)
  File "srsly\msgpack\__init__.py", line 6, in <module>
  File "_packer.pyx", line 7, in init srsly.msgpack._packer
ModuleNotFoundError: No module named 'srsly.msgpack.util'
[26808] Failed to execute script ABC

**Comment by adrianeboyd** [comment]
Hi, it looks like you just need to adjust the hooks for `srsly`. Here's an answer to a similar question that shows how to do this more generally for spacy (you'd need to add `srsly` to this setup): https://stackoverflow.com/a/60360900/461847

It may also be helpful to search for related issues in the issue tracker: https://github.com/explosion/spaCy/issues?q=is%3Aissue+pyinstaller+

**Comment by shikher04** [comment]
@adrianeboyd  tried solving 

> Hi, it looks like you just need to adjust the hooks for `srsly`. Here's an answer to a similar question that shows how to do this more generally for spacy (you'd need to add `srsly` to this setup): https://stackoverflow.com/a/60360900/461847
> 
> It may also be helpful to search for related issues in the issue tracker: https://github.com/explosion/spaCy/issues?q=is%3Aissue+pyinstaller+

Hi @adrianeboyd tried solving with  https://stackoverflow.com/a/60360900/461847 but none of this works for me

**Comment by Zast996** [comment]
@adrianeboyd I tried to follow your solution.
The error persisted, so, following an answer of Mrinal Gupta in the stack overflow link, I appended to hook-spacy.py
` hiddenimports += ['srsly.msgpack.util']`
The error is gone, but there is now a weird problem.
If I execute the exe by double-clicking or from a cmd, there is no error, but the code is not executed (it seems a crash).
If I execute the exe from my Anaconda Prompt, the code is executed succesfully.
The libraries I used in my script are the following:
import spacy
from spacy.gold import biluo_tags_from_offsets
from spacy.tokenizer import Tokenizer
